### ISSUE TITLE
Question for use aceloss

### ISSUE THREAD

**Issue by Shrekbaby** [Luoxd1996/Active_Contour_Euler_Elastica_Loss#5]
Hi, I use the FastACELoss3DV2 as loss function for liver segmentation in nnUNet. Training about 140 epochs  but the global dice for validation data set always below 0.04. Is there any wrong with me in using the loss function?
![aceloss](https://user-images.githubusercontent.com/35718971/127260862-a767b588-7127-4360-86a6-5fd354217857.PNG)

**Comment by Luoxd1996** [comment]
Hi, 
This work was done one and half years ago, I have forgotten some training details. Your log is so strange, maybe you need to check the implementation is right firstly, and then you can combine the ACELoss with dice or CE, a tip is using torch.mean() rather than torch.sum().
Best;
Xiangde.
